CODEOWNERS

```diff
@@ -67,12 +67,12 @@ core/scripts/gateway @bolekk @pinebit
 /operator-ui/ @DeividasK @jkongie
 
 # Contracts
-/contracts/ @se3000 @connorwstein
+/contracts/ @se3000 @connorwstein @RensR
```

Comment:
Will we have more granular code owners on this?

Reply:
Anything below that is more specific overwrites this. I see VRF is missing, but most other teams have their team as owner. Having this top level simply means that anything that falls between the cracks is caught by the people here (e.g. shared folder, which is sorta managed by me)